Formula/pdmenu.rb

```diff
@@ -0,0 +1,34 @@
+class Pdmenu < Formula
+  desc "Full screen menuing system for Unix"
+  homepage "https://joeyh.name/code/pdmenu/"
+  url "http://sources.buildroot.net/pdmenu/pdmenu-1.3.6.tar.gz"
+  sha256 "dedd2a4a5719278b5e49041161990c2f20b5909818837542aaca01819f2c14eb"
+  license "GPL-2.0-only"
+
+  head "git://git.joeyh.name/pdmenu"
+
+  depends_on "gettext" => :build
+  depends_on "s-lang" => :build
+
+  def install
+    ENV.append "CC", "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append "SLANG_H_LOC", "#{HOMEBREW_PREFIX}/include/slang.h"
+    ENV.append "SLANG_LIB_LOC", "#{HOMEBREW_PREFIX}/lib"
+    ENV.append "INSTALL", "#{HOMEBREW_PREFIX}/bin/ginstall"
+
+    # should be fixed in the upcoming 1.3.7
+    system "sed", "-i", "-c", "s?^LIBS.*$?LIBS           = @LIBS@ @INTLLIBS@ $(EFENCE)?", "autoconf/makeinfo.in"
+
+    # we cannot use "/usr/share/locale"
+    system "sed", "-i", "-c", "s?^LOCALEDIR.*$?LOCALEDIR      = $(INSTALL_PREFIX)/#{prefix}/locale?", "Makefile"
+
+    system "./configure", "--prefix=#{prefix}", "--with-libiconv-prefix=#{HOMEBREW_PREFIX}"
+    system "make"
+    mkdir pkgshare.to_s
+    system "make", "install"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/pdmenu -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
pdmenu is a menuing app, I have put the same test I have found in https://github.com/homebrew/homebrew-core/blame/master/Formula/ncdu.rb formula

Reply:
Yes, we have plenty of formulae with bad tests. They were added before we realised they actually weren't reliable tests.

We only add new formulae if they have acceptable tests.